vividus-util/build.gradle

```diff
@@ -7,9 +7,9 @@ dependencies {
     api(group: 'com.fasterxml.jackson.core', name: 'jackson-core')
     api(group: 'com.fasterxml.jackson.core', name: 'jackson-databind')
     api(group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8')
+    api(group: 'org.apache.commons', name: 'commons-lang3', version: versions.commonsLang3)
```

Comment:
this change should go as a separate commit

Reply:
this is related to my changes, otherwise we get compilation error